content/en/account_management/rbac/granular_access.md

```diff
@@ -16,10 +16,11 @@ Use the different principals to control access patterns in your organization and
 | [Dashboards][2]                                  | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
 | [Monitors][3]                                    | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
 | [Notebooks][4]                                   | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
-| [Powerpacks][8]                                  | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
-| [Security rules][5]                              | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
-| [Service Level Objectives][6]                    | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
-| [Synthetic tests][7]                             | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
+| [Powerpacks][5]                                  | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
+| [Security rules][6]                              | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
+| [Security suppressions][7]                       | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
```

Comment:
Just linking to the Suppressions page for now. Will update link when we add the permissions info.